tests/foreman/cli/test_ansible.py

```diff
@@ -561,3 +566,112 @@ def test_positive_install_ansible_collection(
         assert result['success'] == '1'
         collection_path = client.execute('ls ~/ansible_collections').stdout
         assert 'oasis_roles' in collection_path
+
+    @pytest.mark.tier3
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    @pytest.mark.parametrize('auth_type', ['admin', 'non-admin'])
+    def test_positive_ansible_variables_imported_with_roles(
+        self, request, auth_type, target_sat, module_org, module_ak_with_cv, rhel_contenthost
+    ):
+        """Verify that when Ansible roles are imported, their variables are imported simultaneously
+
+        :id: 107c53e8-5a8a-4291-bbde-fbd66a0bb85e
+
+        :steps:
+            1. Register a content host with satellite
+            2. Create a custom role with variables and import it into satellite
+            3. Assert that the role is imported along with associated variables
+            4. Assign that role to a host and verify the role is assigned to the host
+            5. Run the Ansible role
```

Comment:
Can you please update the steps to have more description of the test.

Reply:
I think its descriptive enough, what additional description do you suggest?

Reply:
Maybe we can add more details of where the variables should be placed in the file?

Reply:
It doesn't matter where we keep the variables as its custom role, but I'd cover both the variables directories when mentioned issue is fixed but until then I don't want to keep the steps restricted

Reply:
Ack.